lexicon.txt

```diff
@@ -753,6 +751,7 @@ receiveown
 receivestoreforward
 receivethresholdcontrol
 receivetime
+recognised
```

Comment:
Our spell checker uses [American English](https://en.wikipedia.org/wiki/American_English) spellings, so this should be "recognize".   Let's find it in the source, update it to the US English spelling and delete this.